frontend/src/app/game/pages/intro/intro.component.ts

```diff
@@ -10,6 +10,7 @@ import {GameService} from '../../game.service';
 export class IntroComponent {
   constructor(meta: MetaService, public gameService: GameService) {
     meta.setTitle('Vítejte');
+    changeFavicon();
```

Comment:
Could not this be a default behavior somehow so we have all non-game routes consistent?